.github/workflows/ci.yaml

```diff
@@ -86,10 +89,6 @@ jobs:
         options: --health-cmd "curl -f http://localhost:8080/auth/realms/master" --health-interval 40s --health-retries 8 --health-timeout 60s
 
     steps:
-      - name: Cancel Previous Runs
-        uses: n1hility/cancel-previous-runs@v2
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
While I was at it, I decided to replace this with https://docs.github.com/en/actions/using-jobs/using-concurrency